lib/windows/winlean.nim

```diff
@@ -33,13 +33,15 @@ type
   WINBOOL* = int32
     ## `WINBOOL` uses opposite convention as posix, !=0 meaning success.
     # xxx this should be distinct int32, distinct would make code less error prone
+  PBOOL* = ptr WINBOOL
```

Comment:
how about add `# see https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types` right above this type section?

Reply:
Done